policies/common/ClusterResourceSet.yaml

```diff
@@ -14,13 +15,14 @@ spec:
       apiCall:
         urlPath: "/apis/cluster.x-k8s.io/v1alpha3/namespaces/{{ request.object.metadata.namespace }}/clusters"
         jmesPath: "items[?metadata.name == '{{request.object.metadata.labels.\"cluster.x-k8s.io/cluster-name\"}}'].metadata.labels.\"release.giantswarm.io/version\" | [0]"
-    - name: release
-      apiCall:
-        urlPath: "/apis/release.giantswarm.io/v1alpha1/releases"
-        jmesPath: "items[?metadata.name == 'v{{ releaseVersion }}'] | [0]"
+    preconditions:
+      any:
+      - key: "{{ releaseVersion }}"
+        operator: Equals
+        value: "20.0.0"
```

Comment:
Same as for all other rules - makes it specific to release `20.0.0`

Reply:
What about alpha & beta releases before final GA?

Reply:
I don't think this is necessary as we are mostly using the releases for deployment and as per the RFC we do not upgrade through releases.

So I thought that just `20.0.0` would suffice for now. We can easily extend this to have multiple values here though (if necessary).